[2024-07-04T07:42:44.007+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-07-04T07:42:44.038+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_pipeline.Extraction manual__2024-07-04T07:42:35.704883+00:00 [queued]>
[2024-07-04T07:42:44.049+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_pipeline.Extraction manual__2024-07-04T07:42:35.704883+00:00 [queued]>
[2024-07-04T07:42:44.050+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2024-07-04T07:42:44.073+0000] {taskinstance.py:2330} INFO - Executing <Task(_PythonDecoratedOperator): Extraction> on 2024-07-04 07:42:35.704883+00:00
[2024-07-04T07:42:44.088+0000] {standard_task_runner.py:63} INFO - Started process 270 to run task
[2024-07-04T07:42:44.092+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'dag_pipeline', 'Extraction', 'manual__2024-07-04T07:42:35.704883+00:00', '--job-id', '6', '--raw', '--subdir', 'DAGS_FOLDER/dags.py', '--cfg-path', '/tmp/tmp54v2c8mw']
[2024-07-04T07:42:44.093+0000] {standard_task_runner.py:91} INFO - Job 6: Subtask Extraction
[2024-07-04T07:42:44.145+0000] {task_command.py:426} INFO - Running <TaskInstance: dag_pipeline.Extraction manual__2024-07-04T07:42:35.704883+00:00 [running]> on host b272c9768427
[2024-07-04T07:42:44.244+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='dag_pipeline' AIRFLOW_CTX_TASK_ID='Extraction' AIRFLOW_CTX_EXECUTION_DATE='2024-07-04T07:42:35.704883+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-07-04T07:42:35.704883+00:00'
[2024-07-04T07:42:44.245+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-07-04T07:42:44.247+0000] {extract.py:63} INFO - Starting news extraction process
[2024-07-04T07:42:46.292+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-07-04T07:42:46.293+0000] {taskinstance.py:2905} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/taskinstance.py", line 465, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/taskinstance.py", line 432, in _execute_callable
    return execute_callable(context=context, **execute_callable_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/baseoperator.py", line 401, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/decorators/base.py", line 265, in execute
    return_value = super().execute(context)
                   ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/baseoperator.py", line 401, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/operators/python.py", line 235, in execute
    return_value = self.execute_callable()
                   ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/operators/python.py", line 252, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/airflow/dags/extract.py", line 121, in extract
    p.set_options(p.OPT.MENTION, p.OPT.EMOJI, p.OPT.HASHTAG, p.OPT.RESERVED, p.OPT.SMILEY, p.OPT.URL)
    ^^^^^^^^^^^^^
AttributeError: module 'preprocessor' has no attribute 'set_options'
[2024-07-04T07:42:46.321+0000] {taskinstance.py:1206} INFO - Marking task as FAILED. dag_id=dag_pipeline, task_id=Extraction, run_id=manual__2024-07-04T07:42:35.704883+00:00, execution_date=20240704T074235, start_date=20240704T074244, end_date=20240704T074246
[2024-07-04T07:42:46.347+0000] {standard_task_runner.py:110} ERROR - Failed to execute job 6 for task Extraction (module 'preprocessor' has no attribute 'set_options'; 270)
[2024-07-04T07:42:46.401+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 1
[2024-07-04T07:42:46.434+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-07-04T07:42:46.441+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
